bundles/core/org.eclipse.smarthome.core.test/src/test/java/org/eclipse/smarthome/core/library/types/QuantityTypeTest.java

```diff
@@ -232,6 +233,18 @@ public void testDivide_Zero() {
         new QuantityType<>("4 m").divide(QuantityType.ZERO);
     }
 
+    @Test
+    public void testPpm() {
```

Comment:
The QuantityTypeTest should only test parts of the corresponding class. Please use the SmartHomeUnitsTest for conversion tests.

Reply:
shall the new tests from #5336 be move there, too?